Dizzy/UI Components/LocationLabel.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  LocationLabel.swift
+//  Dizzy
+//
+//  Created by Or Menashe on 4/8/19.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class LocationLabel: UIView {
+    
+    let textLabel = UILabel()
+    let horizontalPadding = CGFloat(4)
+    let cornersRadius = CGFloat(13)
+    let backgroundAlpha = CGFloat(0.5)
+
```

Comment:
Maybe add  horizontalPadding, cornersRadius to Metrics class

Reply:
i am no sure of that because metrics suppose to have values that are valid for al classes.
horizontalPadding is 4 only for this cell, it may be that other components will have different horizontal padding so this is why those are not in Metrics

Reply:
got it